src/components/webpanel/webpanel.ts

```diff
@@ -1,19 +1,18 @@
 import * as vscode from 'vscode';
+import { LogsPanel } from '../logs/logsWebview';
 
 export abstract class WebPanel {
     private disposables: vscode.Disposable[] = [];
     protected content: string;
     protected resource: string;
-    private hasLivePanel = true;
 
     protected static createOrShowInternal<T extends WebPanel>(content: string, resource: string, viewType: string, title: string, currentPanels: Map<string, T>, fn: (p: vscode.WebviewPanel, content: string, resource: string) => T): T {
         const column = vscode.window.activeTextEditor ? vscode.window.activeTextEditor.viewColumn : undefined;
 
         // If we already have a panel, show it.
         const currentPanel = currentPanels.get(resource);
         if (currentPanel) {
-            currentPanel.setInfo(content, resource);
-            currentPanel.update();
+            currentPanel.setContent(content);
```

Comment:
Not sure whether it's safe to remove the resource, or to remove the call to `currentPanel.update()` in the next line.  These might be required for `DescribePanel`.

Reply:
The currentPanel.update() is called inside the `setContent`. It was already there when it was `setInfo`. I just deleted the second call which is useless.